### PR TITLE
Update the website to reflect our new project charter

### PR DESCRIPTION
**Pull request by negz**
Per https://github.com/crossplane/crossplane/pull/3045 we recently defined the project's charter for the first time. This PR updates the website to reflect the new charter.

### CODE REVIEW

**Comment by jbw976** [comment]
@negz it would be great to have a preview available of your changes, so we can get a nice visual of them and also check for any structural/layout issues.

Check out these instructions:https://github.com/crossplane/crossplane.github.io#to-preview-and-share-changes-from-your-fork, so that https://negz.github.io/crossplane.github.io/ will have your latest changes for us to see! :)

File: index.html [hunk 1]
```diff
@@ -99,22 +108,29 @@ <h2>There is a flavor of infrastructure for everyone on Crossplane</h2>
...
-        <h2>The Universal Cloud API</h2>
+        <h2>Control planes - the future of cloud</h2>
         <p>
-          Crossplane provides a consistent API across a diverse set of vendors, resources, and
-          abstractions. The Crossplane Resource Model (XRM) extends the Kubernetes Resource Model
-          (KRM) in an opinionated way, resulting in a uniform experience for managing resources,
-          no matter what provider or vendor built them.
+          The cloud vendors have been building with control planes for years.
+          Now Crossplane helps you do the same. Control planes are
+          self-healing—they automatically correct drift. Consumers can
```
Comment (jbw976) [inline index.html:129]:
```suggestion
          self-healing — they automatically correct drift. Consumers can
```

Comment (jbw976) [reply index.html:129]: spaces around this hyphen/separator? it looks weird to not have spaces when it's smooshed together with the hyphen in self-healing 🤔

File: index.html [hunk 2]
```diff
@@ -61,11 +63,14 @@ <h2>Provision and manage cloud infrastructure and services using kubectl</h2>
...
         <img src="{{ "/images/stacks.png" | relative_url }}" srcset="{{ "/images/stacks.png" | relative_url }} 1x, {{ "/images/stacks@2x.png" | relative_url }} 2x, {{ "/images/stacks@3x.png" | relative_url }} 3x"
           alt="Stacks" id="stacksImg" />
-        <h2>Created to power a more open cloud</h2>
+        <h2>Created to power open platforms</h2>
         <p>
-          We built Crossplane to help organizations build their clouds like the cloud vendors build theirs—with a control plane. Crossplane is a CNCF project which extends the Kubernetes
-          API to manage and compose infrastructure. Operators can encapsulate policies, permissions, and other guardrails behind a custom API line generated by Crossplane while application developers
-          can self-service from the API without needing to become an infrastructure expert.
+          We built Crossplane to help organizations build their platforms like
+          the cloud vendors build theirs—with control planes. Crossplane is an
```
Comment (jbw976) [inline index.html:69]:
```suggestion
          the cloud vendors build theirs — with control planes. Crossplane is an
```

File: index.html [hunk 3]
```diff
@@ -42,12 +44,12 @@ <h1>Compose cloud infrastructure and services into custom platform APIs</h1>
...
       <div class="col-6_md-12">
-        <h2>Provision and manage cloud infrastructure and services using kubectl</h2>
+        <h2>Don’t start from scratch</h2>
         <p>
-          Crossplane extends your Kubernetes cluster, providing you with CRDs for any infrastructure
-          or managed service. Compose these granular resources into higher level abstractions that
-          can be versioned, managed, deployed and consumed using your favorite tools and existing
-          processes you've already integrated with your clusters.
+          The best platforms are built on control planes, but historically
+          building a control plane has required writing code—a lot of code. With
```
Comment (jbw976) [inline index.html:50]:
```suggestion
          building a control plane has required writing code — a lot of code. With
```

File: index.html [hunk 4]
```diff
@@ -42,12 +44,12 @@ <h1>Compose cloud infrastructure and services into custom platform APIs</h1>
...
+          Crossplane you can build a control plane tailored to your unique needs
+          in days, not months, without writing tricky distributed systems code.
```
Comment (jbw976) [inline index.html:52]: "complicated" instead of "tricky"? not a strong feeling on this

**Review by jbw976 — APPROVED** [review]
mostly looks great - want to confirm how things will render (especially if all the icons/graphics make sense for the updated copy).

File: index.html (hunk 4 above)
Comment (negz) [reply index.html:52]: Maybe. I was going more for "hard to get right", which I think "tricky" captures better despite feeling a little informal.

File: index.html (hunk 1 above)
Comment (negz) [reply index.html:129]: I actually copied this "spaceless" pattern from the existing site. See the "Created to power a more open cloud" section.

Comment (jbw976) [reply index.html:129]: ah got it - if that's the existing pattern then that is reasonable :)

Comment (negz) [reply index.html:129]: > it looks weird to not have spaces when it's smooshed together with the hyphen in self-healing

It actually renders out as two different characters when its not in fixed width font. The dash in self-healing is a hyphen while the following one is a longer em-dash.

**Pull request merged**